pkg/api/types.go

```diff
@@ -703,6 +703,9 @@ type TestStepConfiguration struct {
 	// Optional indicates that the job's status context, that is generated from the corresponding test, should not be required for merge.
 	Optional bool `json:"optional,omitempty"`
 
+	// Portable allows to port periodic tests to current and future release despite the demand to skip periodics
+	Portable bool `json:"portable,omitempty"`
```

Comment:
Would it make sense to not copy other job types to those release branches?  If not, we need [validation](https://github.com/openshift/ci-tools/blob/master/pkg/validation/test.go) so it is not added to non-periodic tests.

Reply:
First question: I would like to announce this change and give people the right to keep portability. On non-periodic tests this flag, even enabled makes no effect but it could be added to validation for correctness.
Second question: I will describe it as a part of a branching document.